lib/rubocop/cop/style/operator_method_call.rb

```diff
@@ -50,11 +53,10 @@ def method_call_with_parenthesized_arg?(argument)
           argument.children.first && argument.parent.parenthesized?
         end
 
-        def anonymous_forwarding?(argument)
-          return true if argument.forwarded_args_type? || argument.forwarded_restarg_type?
-          return true if argument.hash_type? && argument.children.first&.forwarded_kwrestarg_type?
+        def forwarding?(argument)
+          return DISALLOWED_ARG_TYPES.include?(argument.children.first&.type) if argument.hash_type?
 
-          argument.block_pass_type? && argument.source == '&'
+          DISALLOWED_ARG_TYPES.include?(argument.type)
```

Comment:
It seems that the duplication with `DISALLOWED_ARG_TYPES.include?` below can be eliminated. However, the combination of the method name `forwarding?` and the constant name `DISALLOWED_ARG_TYPES` might still be a bit unclear.

```ruby
         def forwarding?(argument)
           type = argument.hash_type? ? argument.children.first&.type : argument.type

           DISALLOWED_ARG_TYPES.include?(type)
         end
```

Reply:
Nice suggestion, I like it. Renamed things a bit, so hopefully its clearer now